conf/services/direct-debit.yml

```diff
@@ -19,46 +19,46 @@ accessibilityProblems:
   - the ‘Verify your email’ displays a link as a button, which means the interaction is different to what the user is told
 milestones:
   - description: The ‘Verify your email address’ page does not use an autocomplete feature to allow users to fill this information in automatically. It also does not display the correct keyboard for email addresses to be entered easily. This doesn't meet WCAG success criterion 1.3.5 (Identify Input Purpose).
-    date: 2020-07-31
+    date: 2021-12-15
   - description: |
       The service has errors in the order of elements, attribute syntax, and
       incorrectly nested start and end tags which means assistive technologies
       cannot use the service reliably. This does not meet WCAG success criterion
       4.1.1 (Parsing).
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       The ‘change’ links on the ‘Check your details are correct’ page are not
       descriptive enough and have the same name. This does not meet WCAG 2.1
       success criterion 2.4.4 (Link Purpose (Link only)).
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       On the ‘Verify your email’ page there is collapsible content which may
       cause issues for people using a screen reader. This does not meet WCAG
       success criterion 4.1.2 (Name, Role, Value).
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       On the ‘Which email do you want to use?’ page there is a missing legend,
       so the purpose of the radio buttons may be unclear for some users. This
       does not meet WCAG success criterion 1.3.1 (Info and Relationships).
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       The service ends the user’s session if they have been idle for 15 minutes,
       however it does not give the user the chance to extend their session. This
       does not meet WCAG success criteria 2.2.1 (Timing Adjustable).
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       The hint text on the ‘Enter the new email address’ field is not associated
       with the fieldset, this means if a screen-reader user moves directly to
       the radio inputs they will not hear the hint text. This does not meet WCAG
       success criterion 2.4.6 (Headings and Labels).
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       On the ‘Verify your email’ page a button is used when displaying the link
       ‘Send it again’. This means the interaction is different to what the user is
       told, either visually or via a screen-reader. This button also does not
       have a focus state which means it is not compatible with keyboard navigation.
       This does not meet WCAG success criterion 2.4.7 (Focus Visible).
-    date: 2020-09-30
+    date: 2021-12-15
 serviceLastTestedDate: 2020-07-06
 statementVisibility: public
 statementCreatedDate: 2020-06-30
```

Comment:
Does the `statementLastUpdatedDate` need to be updated here?

Reply:
Yes